docs/console.html

```diff
@@ -0,0 +1,804 @@
+                        <pre class="prettyprint"><code class="language-javascript">
+function connectToServer() {
+    console.error("Error: %s (%i)", "Server is  not responding",500);
+}
+connectToServer();
+</code></pre>
+                    </figure>
+                    <figure>
+                        <img src="console-files/error-server-not-resp.png" alt="Error example output"/>
+                        <figcaption>
+                           How <code>connectToServer()</code> displays in the console.
+                        </figcaption>
+                    </figure>
+                </figure>
+                
+                <figure>
+                    <figcaption>Using the <code>warn()</code> method.</figcaption>
+                    <figure>
+                        <pre class="prettyprint"><code class="language-javascript">
+if(a.childNodes.length < 3 ) {
+    console.warn('Warning! Too few nodes (%d)', a.childNodes.length);
+}
+</code></pre>
+                    </figure>
+                    <figure>
+                        <img src="console-files/warning-too-few-nodes.png" alt="Warn example" />
+                        <figcaption>
+                            Example warning output.
+                        </figcaption>
+                    </figure>
+                </figure>
+            </figure>
+
+        </section>
+
+        <section>
+            <h3 id="assertions">Assertions</h3>
+            <p>
+                The <code><a href="console-api#consoleassertexpression-object">console.assert()</a></code> method conditionally displays an error string (its second parameter) only if its first parameter evaluates to <samp>false</samp>.
+            </p>
+            <figure>
+                <figcaption>A simple assertion and how it displays.</figcaption>
+                <figure>
+                    <figcaption>The following code will cause an error message in the console only if the number of child nodes belonging to the <code>list</code> element is greater than 500.</figcaption>
+                    <figure>
+                        <pre class="prettyprint"><code class="language-javascript">
+console.assert(list.childNodes.length < 500, "Node count is > 500");              
+</code></pre>
+                    </figure>
+                </figure>
+                <figure>
+                    <img src="console-files/assert-failed.png" alt="Assertion failed" />
+                    <figcaption>
+                        How an assertion failure displays in the console.
+                    </figcaption>
+                </figure>
+            </figure>
+        </section>
+
+        <section>
+            <h3 id="filtering-console-output">Filtering console output</h3>
+            <p>
+                You can filter console output by its severity level by selecting one of the filter options.
+                Filters are enabled under the filter funnel icon located in the upper-left corner of the console panel.
```

Comment:
regex is brand new. gotta mention that.

also negation i believe works in console. big request.